tmva/pymva/src/MethodPyKeras.cxx

```diff
@@ -231,6 +231,21 @@ void MethodPyKeras::SetupKerasModel(bool loadTrainedModel) {
     * Load Keras model from file
     */
 
+   Log() << kINFO << " Setup Keras Model " << Endl;
+
+   PyRunString("load_model_custom_objects=None");
```

Comment:
There is no documentation that describes how the feature works. It must be explained somewhere that the external file requires an object with name `load_model_custom_objects`, which is a dictionary, that the loading works.

Reply:
Also the object has to be in global scope!

Reply:
Hi Stefan, 
Actually I have verified the user object, load_model_custom_objects, must be in the local namespace defined by the method, otherwise it will not work.

The problem is in the user function implementations. They are in a different namespace, and in the example above K = keras.backend must be defined in the global namespace. 
A possible solution is to define the  lines importing keras and setting K in global namespace to make it work or to execute user code in both local and global namespace. 
This also works.